src/interpreter/plugin/3rdparty/bessel/bessel.ts

```diff
@@ -0,0 +1,441 @@
+/* eslint-disable */
```

Comment:
We should remove this line and all `@ts-ignore` and add `/src/interpreter/plugin/3rdparty/**` to ESLint and TSLint excludes.

Reply:
what is the rationale for removing @ts-ignore? I want to have some typing in those files where I can, and just ignore in few places

Reply:
Right. Public API may be helpful 👍